vulkano/src/pipeline/graphics/mod.rs

```diff
@@ -2572,6 +2573,130 @@ impl GraphicsPipelineCreateInfo {
             }
         }
 
+        if match device
+            .physical_device()
+            .properties()
+            .conservative_point_and_line_rasterization
+        {
+            Some(b) => !b,
+            None => true,
+        } {
```

Comment:
```suggestion
        let properties = device.physical_device().properties();

        if matches!(conservative_rasterization_state.mode, ConservativeRasterizationMode::Disabled)
            && !properties.conservative_point_and_line_rasterization.unwrap_or(false)
        {
```

And then all the matches for `ConservativeRasterizationMode::Disabled` further below are no longer needed.